modules/claims_api/app/controllers/claims_api/base_form_controller.rb

```diff
@@ -76,5 +80,14 @@ def format_526_errors(errors)
         { status: 422, detail: "#{error['key']} #{error['detail']}", source: error['key'] }
       end
     end
+
+    def track_526_validation_errors(errors)
```

Comment:
can you include a spec test for this? I can see someone trying to change this in the future.